src/components/BadgeSelect.tsx

```diff
@@ -99,7 +103,7 @@ class BadgeSelect extends React.Component<BadgeSelectProps, BadgeSelectState> {
 
 				{this.props.items.map(item => (
 					<ButtonWrapper key={item} onClick={() => this.setSelected(item)}>
-						<Badge text={item} />
+						{<Badge text={item} />}
```

Comment:
Why is this needed?

Reply:
Not needed, probably just remained after doing something else.